api/probe.go

```diff
@@ -22,4 +22,6 @@ type KogitoProbeInterface interface {
 	SetLivenessProbe(livenessProbe corev1.Probe)
 	GetReadinessProbe() corev1.Probe
 	SetReadinessProbe(readinessProbe corev1.Probe)
+	GetStartupProbe() corev1.Probe
+	SetStartupProbe(startupProbe corev1.Probe)
```

Comment:
@Kevin-Mok can you please also open a PR to downstream operator and implement the interface there? Since we are updating the interface, we also need to update product CRDs. Thanks!

Reply:
README? what README, @spolti ? Maybe the docs? Yes, the docs would be a good fit.

Reply:
@ricardozanini sry, I meant about the need to replicate the interface methods, not about the feature itself..

Reply:
@Kevin-Mok did you create the PR to update the rhpam-kogito-operator?

Reply:
I just opened the [downstream PR](https://github.com/kiegroup/rhpam-kogito-operator/pull/12).